src/pages/panel/views/main.js

```diff
@@ -53,17 +53,29 @@ function reloadTab(host, event) {
 }
 
 async function togglePause(host, event) {
+  host.alert = '';
+
   const { paused, pauseType } = event.target;
+  const { options, stats } = host;
 
-  host.alert = '';
+  if (paused) {
+    const pausedHostname = Object.keys(options.paused)
+      .sort()
+      .reverse()
```

Comment:
no need to reverse if you change the sorting function

Reply:
Yes, but that is more readable - I don't like sorting function like `b - a`, which indicates nothing meaningful. The `sort` and `reverse` methods are highly optimized, and this is a callback of user action, so the performance of the execution is not an issue.